src/main/java/org/apache/commons/lang3/SerializationUtils.java

```diff
@@ -122,7 +122,7 @@ public SerializationUtils() {
      * @since 3.3
      */
     public static <T extends Serializable> T roundtrip(final T msg) {
-        return SerializationUtils.deserialize(SerializationUtils.serialize(msg));
+        return (T) SerializationUtils.deserialize(SerializationUtils.serialize(msg));
```

Comment:
Which JDK are you using? What is the output of ```mvn -v```on your system? It should build with maven 3 and JDK 1.6. If it doesn't I'd like to fix this problem in a separate PR, since this change is unrelated to the problem described in LANG-1081.